### DISCUSSION TITLE
Force completion window to show always under current line

### DISCUSSION THREAD

**Discussion by ADIX7** [hrsh7th/nvim-cmp#1967] [General]
Is there a way to force the completion window to always open under the current line? I also use https://github.com/ray-x/lsp_signature.nvim and when the cmp window shows above the current line, it  covers the signature help window.

**Comment by Shougo** [comment]
Please see this.

https://github.com/hrsh7th/nvim-cmp/issues/495

**Comment by ADIX7** [comment]
Is there a way to get if the cmp window is currently open and if it isbelow/above the current line?
